### ISSUE TITLE
Error spinning up new Docker image with Mojo > 9

### ISSUE THREAD

**Issue by PanopticaRising** [drdrew42/renderer#45]
Running the Dockerfile now fails with the following error:

```home directory /usr/app/lib
Can't load application from file "/usr/app/script/render_app": Route pattern "/webwork2_files/*path" contains a reserved stash value at /usr/local/share/perl/5.30.0/Mojolicious/Routes/Route.pm line 222.
        Mojolicious::Routes::Route::_route(Mojolicious::Routes=HASH(0x55c47cde1098), "/webwork2_files/*path") called at /usr/local/share/perl/5.30.0/Mojolicious/Routes/Route.pm line 197
        Mojolicious::Routes::Route::_generate_route(Mojolicious::Routes=HASH(0x55c47cde1098), ARRAY(0x55c480a8c4b0), "/webwork2_files/*path", CODE(0x55c480a8bf10)) called at /usr/local/share/perl/5.30.0/Mojolicious/Routes/Route.pm line 39
        Mojolicious::Routes::Route::any(Mojolicious::Routes=HASH(0x55c47cde1098), "/webwork2_files/*path", CODE(0x55c480a8bf10)) called at /usr/app/lib/RenderApp.pm line 90
        RenderApp::startup(RenderApp=HASH(0x55c47e47a868)) called at /usr/local/share/perl/5.30.0/Mojolicious.pm line 164
        Mojolicious::new("RenderApp") called at /usr/local/share/perl/5.30.0/Mojo/Server.pm line 18
        Mojo::Server::build_app(Mojo::Server=HASH(0x55c47e47a760), "RenderApp") called at /usr/local/share/perl/5.30.0/Mojolicious/Commands.pm line 82
        Mojolicious::Commands::start_app("Mojolicious::Commands", "RenderApp") called at /usr/app/script/render_app line 11
        require /usr/app/script/render_app called at (eval 64) line 1
        eval 'package Mojo::Server::Sandbox::5e71eca6a9880a56092744e063afc268; require $path' called at /usr/local/share/perl/5.30.0/Mojo/Server.pm line 56
        Mojo::Server::load_app(Mojo::Server::Prefork=HASH(0x55c47ca69f38), "/usr/app/script/render_app") called at /usr/local/share/perl/5.30.0/Mojo/Server/Hypnotoad.pm line 51
        Mojo::Server::Hypnotoad::run(Mojo::Server::Hypnotoad=HASH(0x55c47ca4a650), "/usr/app/script/render_app") called at /usr/local/bin/hypnotoad line 14
Compilation failed in require at (eval 64) line 1.
```

The only references I've found were [this issue](https://github.com/preaction/Mojolicious-Plugin-Moai/issues/2#issue-823962645) which seems accurate to the cause of the problem. I can start the Dockerfile with the following patch as a workaround:

```diff --git a/Dockerfile b/Dockerfile
index 468810cd..c1527fa4 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -35,7 +35,7 @@ RUN apt-get update \
     && apt-get clean \
     && rm -fr /var/lib/apt/lists/* /tmp/*
 
-RUN cpanm install Mojo::Base Statistics::R::IO::Rserve Date::Format Future::AsyncAwait \
+RUN cpanm install https://cpan.metacpan.org/authors/id/S/SR/SRI/Mojolicious-8.73.tar.gz Statistics::R::IO::Rserve Date::Format Future::AsyncAwait \
     && rm -fr ./cpanm /root/.cpanm /tmp/*
 
 ENV MOJO_MODE=production```

**Comment by PanopticaRising** [comment]
I also tried using `cpanm`'s versioning syntax to specify 8.73 or something less than 9 but it kept failing.

**Comment by Miniland1333** [comment]
Could you please verify that you are working on the current version of the master branch? A fix was deployed a while ago at 0cd386e8e6f4c643fe39d87f5f612b7d280f5eda to address that issue with Mojo 9.

**Comment by drdrew42** [comment]
Correct, Mojolicious 9 [added protected stash values](https://mojolicious.io/blog/2021/02/14/announcing-mojolicious-9-0/). This fix is in the master branch now.
